src/pages/home/_what-our-clients-say.js

```diff
@@ -1,247 +1,165 @@
-            maxWidth: '800px',
-            margin: '0 auto',
-        },
-        slide_style: {
-            minWidth: '100%',
-            paddingLeft: '1rem',
-            position: 'relative',
-        },
-        chevron_style: {
-            chevron_color: 'black',
-        },
-    }
-
+    const { icon } = getRatingData(4.5)
     return (
-        <SectionContainer padding="5rem 0 0">
-            <Container direction="column">
-                <Header align="center" as="h3" type="section-title">
-                    {localize('What our clients say about Deriv')}
-                </Header>
-            </Container>
-            <Carousel has_autoplay autoplay_interval={6000} {...settings}>
-                {our_client_slides.map((trader, idx) => (
-                    <div key={idx}>
-                        <ClientSlide
-                            key={trader.name}
-                            quote={trader.quote}
-                            name={trader.name}
-                            location={trader.location}
-                            img={data[trader.id]}
-                            img_alt={trader.name + localize(" - Deriv's Client")}
-                        />
-                    </div>
-                ))}
-            </Carousel>
-        </SectionContainer>
+        <StyledContainer>
+            <ClientContainer padding="5rem 0 0">
+                <LineTab>
+                    <ClientFlex jc="space-between" ai="center">
+                        <ClientCard className="left" direction="column">
+                            <Header as="h2" type="heading-2">
+                                {localize('What our clients say about Deriv')}
+                            </Header>
+                            <TrustPilotWidget />
+                        </ClientCard>
+                        <ClientCard className="right" direction="column">
+                            <Header as="h3" type="heading-3">
+                                <Localize
+                                    translate_text="Selling has been <0>seamlessly</0> done and it makes weekend withdrawals an additional plus for this app!"
+                                    components={[<span className="emphasis" key={0} />]}
+                                />
+                            </Header>
+                            <Text size="2.4rem" className="dp2p-text">
+                                {localize('- DP2P apps')}
+                            </Text>
+                            <AppDownloadBox>
+                                <AppDownloadBtns>
+                                    <AppLink
+                                        href="https://play.google.com/store/apps/details?id=com.deriv.dp2p"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                    >
+                                        <img src={GooglePlay} />
```

Comment:
Add alt tag for all images

Reply:
Note that only images with no supporting text will need alts with descriptions. For icons that act as supplementary elements to beautify the section, those will need empty alt tags.